addons/website_version/views/marketing_view.xml

```diff
@@ -0,0 +1,241 @@
+                    <field name="website_id"/>
+                    <field name="state"/>
+                    <field name="sequence" widget="handle"/>
+                </tree>
+            </field>
+        </record>
+
+        <record id="experiment_search" model="ir.ui.view">
+            <field name="name">experiment.search</field>
+            <field name="model">website_version.experiment</field>
+            <field name="arch" type="xml">
+                <search string="Search Experiment">
+                    <field name="website_id"/>
+                    <field name="state"/>
+                    <group string="Group By">
+                        <filter string="Website" domain="[]" name="my_exps" context="{'group_by':'website_id'}"/>
+                        <filter string="State" domain="[]" context="{'group_by':'state'}"/>
+                    </group>
+                </search>
+            </field>
+        </record>
+
+        <!-- <record id="action_experiment_search" model="ir.actions.act_window">
+            <field name="name">Experiment search</field>
+            <field name="res_model">website_version.experiment</field>
+            <field name="view_mode">list</field>
+            <field name="context">{'search_default_my_exps':1}</field>
+        </record> -->
+
+        <record model="ir.ui.view" id="experiment_version_form_view">
+            <field name="name">experiment_version.form</field>
+            <field name="model">website_version.experiment_version</field> 
+            <field name="arch" type="xml">
+                <form string="Experiment_page Form">
+                    <notebook colspan="4">
+                        <page string="Features" colspan="4" >
+                            <field name="version_id" domain="[('website_id','=',context.get('website_id'))]" colspan="4"/>
+                            <field name="frequency"/>
+                        </page>
+                    </notebook>
+                </form>
+            </field>
+        </record>
+            
+        <record id="action_version" model="ir.actions.act_window">
+            <field name="name">Versions</field>
+            <field name="res_model">website_version.version</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+        </record>
+
+        <record model="ir.ui.view" id="version_form_view">
+            <field name="name">version.form</field>
+            <field name="model">website_version.version</field> 
+            <field name="arch" type="xml">
+                <form string="version Form">
+                    <sheet string="Snap">
+                        <group>
+                            <button icon="fa-pencil" name="action_publish" string="Publish" type="object" class="oe_stat_button pull-right" 
+                                context="{'active_id': active_id}"/>
```

Comment:
Actually this is not necessary. Remove the context hack, action_publish will receive the ids.